SpeedyContactAdd WatchKit Extension/Controller/InterfaceController.swift

```diff
@@ -0,0 +1,202 @@
+    }
+    @IBAction func btnNumber7Action() {
+        numberAdd(myCharacter: "7")
+    }
+    @IBAction func btnNumber8Action() {
+        numberAdd(myCharacter: "8")
+    }
+    @IBAction func btnNumber9Action() {
+        numberAdd(myCharacter: "9")
+    }
+    @IBAction func btnNumber0Action() {
+        numberAdd(myCharacter: "0")
+    }
+    @IBAction func btnBackspaceAction() {
+        if numberStore != "" {
+            numberStore.remove(at: numberStore.index(before: numberStore.endIndex))
+            lblPhoneNumber.setText("\(numberStore)")
+        } else {return}
+    }
+    // MARK: - Force Push button resets all
+    @IBAction func btnMenuClear() {
+        clearItems()
+    }
+    
+    // MARK: - Add name to RecipNameStrings
+    @IBAction func btnAddNameAction() {
+        presentTextInputController(withSuggestions: ["New Contact"], allowedInputMode: WKTextInputMode.plain) { (result) in
+            guard let result = result else {return}
+            let resultString = result[0] as! String
+            self.recipNameString = resultString
+            self.lblReName.setText("\(resultString)")
+        }
+    }
+    
+    
+    
+    //  MARK: Send user info to iPhone. If either name or number are blank, do nothing.
+    @IBAction func btnSendtoPhone() {
+        
+        if recipNameString == "" || numberStore == "" {return}
+        
+        // TODO: - Add in a notification that states one of these is blank.
+        
+        if (WCSession.default.isReachable) {
+             if checkIfUserDefaultExist("SavedContacts") == true {
+                loadStoredContacts()
+            }
+            sendContactsToPhone()
+            DispatchQueue.main.async {
+                self.notifyUserAfterSave(inRange: true,
+                                         UUID: UUID().uuidString,
+                                         contactName: self.recipNameString)
+            }
+        } else {
+            //   If WCSession isn't reachable, store contact on watch until phone is reachable.
+            storeSavedContacts()
+            DispatchQueue.main.async {
+                self.notifyUserAfterSave(inRange: false,
+                                         UUID: UUID().uuidString,
+                                         contactName: self.recipNameString)
+            }
+        }
+        // Reset Strings for next contact upload
+        self.clearItems()
+    }
+    
+    // MARK: Sends data to phone
+    func sendContactsToPhone() {
+        let location = getLocation()
+        let contactData = Contacts.init(name: recipNameString,
+                                        phoneNumber: numberStore,
+                                        latitude: location.0,
+                                        longitude: location.1,
+                                        date: location.2)
+        contacts.append(contactData)
+        let encoder = JSONEncoder()
+        let data = (try? encoder.encode(contacts))!
```

Comment:
Does:
```suggestion
        let data = try! encoder.encode(contacts)
```
... not work?